src/main/webapp/WEB-INF/tags/admin/accounts/accountTable.tag

```diff
@@ -0,0 +1,90 @@
+                    <tr class="accountEntry">
+                        <td>
+                            <span class="bold">Google ID: </span>
+                            <a href="${row.instructorHomePageViewLink}" target="blank">${row.account.googleId}</a>
+                            <br>
+                            <span class="bold">Name: </span>${row.account.name}
+                            <br>
+                            <span class="bold">Email: </span>${row.account.email}
+                        </td>
+                        <td id="courses_${row.account.googleId}">
+                            <c:choose>
+                                <c:when test="${not empty row.instructorList}">
+                                    Total Courses: ${fn:length(row.instructorList)}<br>
+                                    <c:forEach items="${row.instructorList}" var="instructor">
+                                        --- ${instructor.courseId}<br>
+                                    </c:forEach>
+                                </c:when>
+                                <c:otherwise>
+                                    No Courses found
+                                </c:otherwise>
+                            </c:choose>
+                        </td>
+                        <td id="${row.account.googleId}_institude">${row.account.institute}</td>
+                        <td id="${row.account.googleId}_createAt">${row.createdAt}</td>
+                        <td>
+                            <a class="btn btn-link btn-xs" id="${row.account.googleId}_details"
+                                href="${row.adminViewAccountDetailsLink}" target="blank">
+                                <span class="glyphicon glyphicon-info-sign"></span> View Details
+                            </a>
+                            &nbsp;&nbsp;&nbsp;&nbsp;
+                            <a class="btn btn-link btn-xs" id="${row.account.googleId}_delete"
+                                href="${row.adminDeleteInstructorStatusLink}" role="button">
+                                <span class="glyphicon glyphicon-remove"></span> Delete Instructor Status
+                            </a>
+                            <a class="btn btn-link btn-xs" id="${row.account.googleId}_deleteAccount"
+                                href="row.adminDeleteAccountLink"
+                                onclick="return toggleDeleteAccountConfirmation('${row.account.googleId}')"> 
+                                <span class="glyphicon glyphicon-trash"></span> Delete Entire Account
+                            </a>
+
+                            <form method="post" target="blank"
+                                action="<%=Const.ActionURIs.ADMIN_ACTIVITY_LOG_PAGE%>">
+                                <button type="submit" 
+                                    id="${row.account.googleId}_recentActions"
+                                    class="btn btn-link btn-xs">
```

Comment:
Indent id and class to be below type.

<button type="submit"
             id="..."
             class="...">